am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -39,9 +39,11 @@ void whenCreatingResourceAccessResourceAccessAppearsInDatabase() {
 
     @Test
     void whenCreatingResourceAccessMultipleEntriesAppearInDatabase() {
-        Map<JsonPointer, Set<Permission>> multipleAttributePermissions = new ConcurrentHashMap<>();
-        multipleAttributePermissions.put(JsonPointer.valueOf(""), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS);
-        multipleAttributePermissions.put(JsonPointer.valueOf("/name"), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS);
+        Map<JsonPointer, Set<Permission>> multipleAttributePermissions =
+            ImmutableMap.<JsonPointer, Set<Permission>>builder()
```

Comment:
Using `#of` is bit better because it can nicely interfere type, something that builder cannot.

I'd suggest using `#of` method in favour of builder in simple use cases.

Reply:
👍